docs/other_integrations.rst

```diff
@@ -59,3 +59,10 @@ Requests
 --------
 
 .. automodule:: ddtrace.contrib.requests
+
+.. _grpc:
+
+Grpc
+^^^^
```

Comment:
The others are doing `------` any reason to do `^^^^`?

Reply:
doing `^^^^` will actually nest it under the `Requests` section, which we do not want.